README.md

```diff
@@ -79,7 +79,28 @@ rocker {
   }
 }
 
-rockerVersion = '1.2.2'  // optional
+rockerVersion = '1.3.0'  // optional
+```
+
+This is a sample configuretion in Kotlin DSL:
+
+```
+import nu.studer.gradle.rocker.RockerConfig
+
+plugins {
+    id("nu.studer.rocker") version "1.0.1"
+    id("java")
+}
+
+configure<NamedDomainObjectContainer<RockerConfig>>() {
```

Comment:
The usage of the plugins block above make it possible to use type safe generated accessors.
Also, the import above is not needed then.

```suggestion
rocker {
```